pyverdict/pyverdict/jvm.py

```diff
@@ -0,0 +1,17 @@
+import time
+import sys
+import subprocess
+from py4j.java_gateway import JavaGateway
+import pyverdict
+
+class JVM:
```

Comment:
Put this logic inside VerdictContext so that the jvm connetion is made when VerdictContext is instantiated.

Reply:
Set the gateway as a class variable so that if there is an existing gateway created when instantiating another VerdictContext, the gateway is shared.